### PR TITLE
chore(react-native): update react native from 0.64.4 to 0.67.5 [CONFIA-1204]

### PR DESCRIPTION
**Pull request by pedrohamarques**
# What

Updates React Native from 0.64.4 to 0.67.5.

Changelogs:
0.65.0 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0650
0.65.1 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0651
0.66.0 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0660
0.66.1 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0661
0.66.2 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0662
0.66.3 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0663
0.66.4 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0664
0.66.5 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0665
0.67.0 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0670
0.67.1 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0671
0.67.2 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0672
0.67.3 -https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0673
0.67.4 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0674
0.67.5 - https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0675

# Why

To keep our version as up to date as possible

# How
Using https://react-native-community.github.io/upgrade-helper/?from=0.64.4&to=0.67.5 website.

# Sample

# QA

1. Run `yarn` on your project root;
2. Run `cd ios && pod install` to update your Podfile. In case there's some error about updating RCT-Folly, run `pod update` in ios folder;
3. Run both Android and iOS emulators;
4. Run `yarn storybook` and check if it is working properly.

[CONFIA-1204](https://produtomagnetis.atlassian.net/browse/CONFIA-1204)

[CONFIA-1204]: https://produtomagnetis.atlassian.net/browse/CONFIA-1204?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: android/app/build.gradle
Comment (lucianomlima) [inline android/app/build.gradle:1]: 🔧 Faltou remover o compileOptions daqui
https://github.com/magnetis/astro-native/pull/720/files#diff-9526ccfd1d1813ed49c39f8c54dbeb512607376a007d824b905bc8b4e4d202d9R132-R135

File: android/build.gradle
Comment (lucianomlima) [inline android/build.gradle:1]: 🔧 Faltou remover o mavenLocal daqui
https://github.com/magnetis/astro-native/pull/720/files#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aR24

File: .gitignore
Comment (lucianomlima) [inline .gitignore:1]: 🔧 Está faltando o arquivo novo `.bundle/config` que não deve ter sido incluído por causa do .gitignore (aconteceu o mesmo na atualização do mobile-app).

Comment (pedrohamarques) [reply .gitignore:1]: Corrigido em [e1986a2](https://github.com/magnetis/astro-native/pull/720/commits/e1986a2fac0e71ae1b70cbf8a37d61829ee17305)

File: android/build.gradle
Comment (pedrohamarques) [reply android/build.gradle:1]: Corrigido em [e1986a2](https://github.com/magnetis/astro-native/pull/720/commits/e1986a2fac0e71ae1b70cbf8a37d61829ee17305)

File: android/app/build.gradle
Comment (pedrohamarques) [reply android/app/build.gradle:1]: Corrigido em [e1986a2](https://github.com/magnetis/astro-native/pull/720/commits/e1986a2fac0e71ae1b70cbf8a37d61829ee17305)

**Review by lucianomlima — APPROVED** [review]

**Review by pedro-cerdera — APPROVED** [review]

**Pull request merged**